src/aws/osml/tile_server/viewpoint/routers.py

```diff
@@ -177,28 +177,22 @@ def create_viewpoint(viewpoint_request: ViewpointRequest, response: Response) ->
             return db_response
 
         @api_router.delete("/{viewpoint_id}")
-        def delete_viewpoint(viewpoint_id: str) -> ViewpointModel:
+        def delete_viewpoint(viewpoint_id: str) -> Dict[str, str]:
```

Comment:
Any reason we can't return a well defined type here? Last time I checked the FastAPI code was using these data classes for things like generating OpenAPI specifications etc.

Reply:
Will update to have respond with empty 204.